test/Riok.Mapperly.Tests/Mapping/ObjectPropertyNullableTest.cs

```diff
@@ -29,6 +51,71 @@ public void NullableIntToNonNullableIntProperty()
             );
     }
 
+    [Fact]
+    public void NullableIntToNullableIntPropertyWithNoNullAssignment()
```

Comment:
Test name should be `NullableIntToNonNullableIntPropertyWithNoNullAssignment`.

Reply:
Fixed